people/Eliška.json

```diff
@@ -0,0 +1,11 @@
+{
+  "key": "eliska",
+  "name": "Eliška Wichová",
+  "nick": "Eliska",
+  "email": "eliska.wichova@mild.blue",
+  "title_cs": "OPERATIONS MANAGER",
+  "title_en": "OPERATIONS MANAGER",
+  "short_desc_en": "Eliška holds a Master's degree in Landscape Engineering. She has various experience from IT and banking corporate. She is the person who ensures the smooth running of Mild Blue´s back office and business processes.",
```

Comment:
neni to corporation? 

a mozna from IT and banking corporations by to melo byt?